packages/dataviews/src/types/field-api.ts

```diff
@@ -92,6 +92,80 @@ export type Rules< Item > = {
 		  ) => Promise< null | string > );
 };
 
+export type NormalizedRules< Item > = {
+	required:
+		| false
+		| {
+				constraint: any;
+				validate: (
+					item: Item,
+					field: NormalizedField< Item >
+				) => boolean;
+		  };
+	elements:
+		| false
+		| {
+				constraint: any;
+				validate: (
+					item: Item,
+					field: NormalizedField< Item >,
+					elements: Array< { value: any } >
```

Comment:
Why do we need `elements` here? Can we create a separate type to reuse, as is the same for every rule (besides the elements here)